src/app/questionGenerator.js

```diff
@@ -0,0 +1,44 @@
+class QuestionGenerator {
+
+    //argument argAmountOfId will be relaced by connection to CurrentMode and it's IDs'
+    static generateQuestionsForCurrentMode(amountOfId) {
+        const amountOfAnswersPerQuestion = 4;
+        let generatedIds = [];
+
+        if (amountOfId < amountOfAnswersPerQuestion) {
```

Comment:
Error handling is really important! Great that you thought about that. What would you want the app to do in this situation? Thinking futher (not necessarily in this task) - how should the UI behave in this case?

Reply:
I was thinking about that - I think we should have more things done in project to decide what should happen here.
I'll keep in mind!

Reply:
Definitely! Controlled crash is a good crash :)